packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -250,7 +250,9 @@ export class ShortcutItem extends React.Component<
         {this.isLocationBeingEdited(index)
           ? this.getShortCutAsInput(binding, index)
           : this.getShortCutForDisplayOnly(binding)}
-        {this.getOrDiplayIfNeeded(index < nonEmptyBindings.length - 1)}
+        {!(index===this._nonEmptyBindings.length-1 &&
```

Comment:
Thanks! Just leaving a note that the linter job is failing on CI with:

```
+ jlpm run prettier:check
Checking formatting...
[warn] packages/shortcuts-extension/src/components/ShortcutItem.tsx
[warn] Code style issues found in the above file. Run Prettier to fix.
+ echo 'Please run `jlpm run prettier` locally and push changes'
+ exit 1
Please run `jlpm run prettier` locally and push changes
```

I think this is because of missing spaces around `===` and `-` but there may be more small reformatting needed

Reply:
@krassowski Yes, the issue was related to code formatting, and I resolved it.